sources/3.0.0/sections/04-data_content_struct.adoc

```diff
@@ -157,12 +157,12 @@ The class *CV_GridEnvelope* provides the grid coordinate values for the diametri
 [level=7]
 ===== low
 
-The attribute _low_ shall be the minimal coordinate values for all grid points within the envelope. For this specification this represents the Southwestern coordinate.
+The attribute _low_ shall be the minimal coordinate values for all grid points within the envelope. For this specification this represents the Southwestern corner coordinate of the southwesternmost grid cell.
```

Comment:
A clarification of some sort might be useful. As grid points are in the center of the cell this would be grid point coordinate minus half grid spacing in both X- and Y-directions.

Reply:
That sounds like a double minus/plus to me. To me personally the formatting is clear as it is, but if we'd like to have this clarified further we should probably write something like:

[Low] ...for this specification the value is southwesternmost _grid point_ coordinate _minus half grid spacing_ for each coordinate axis

[High] ...for this specification the value is northeasternmost _grid point_ coordinate _plus half grid spacing_ for each coordinate axis

What do you think?

Reply:
I agree with @tfilppula in general but would like to propose a very slight change to improve readability. Including the context in @AnnaWall01 's suggestion above:

The attribute _low_ shall be the minimum coordinate values for each coordinate axis among all grid cells within the envelope. For this specification the value is the southwesternmost grid point coordinate minus half of the grid spacing for each coordinate axis.

The attribute _high_ shall be the maximum coordinate values for each coordinate axis among all grid cells within the envelope. For this specification the value is the northeasternmost grid point coordinate plus half of the grid spacing for each coordinate axis.

Reply:
Changed according to @hasel001 proposal.